src/main/interface/settings.c

```diff
@@ -909,6 +909,7 @@ const clivalue_t valueTable[] = {
     { "p_yaw",                      VAR_UINT8  | PROFILE_VALUE, .config.minmax = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, pid[PID_YAW].P) },
     { "i_yaw",                      VAR_UINT8  | PROFILE_VALUE, .config.minmax = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, pid[PID_YAW].I) },
     { "d_yaw",                      VAR_UINT8  | PROFILE_VALUE, .config.minmax = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, pid[PID_YAW].D) },
+    { "true_f_yaw",                 VAR_UINT8  | PROFILE_VALUE, .config.minmax = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, yaw_true_ff) },
```

Comment:
```suggestion
    { "yaw_true_ff",                 VAR_UINT8  | PROFILE_VALUE, .config.minmax = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, yaw_true_ff) },
```

Reply:
Oh... now I understand what you wanted to do, recall the "i_yaw", "d_yaw" etc... but could we just call it ff_yaw or f_yaw and add a pid[PID_YAW].F or pid[PID_YAW].FF field? I didn't find a way to simulate BF's FF yet. Thinking about future implementations so we would avoid breaking changes in naming

Reply:
I suppose that we still have a pid.f in the code that this could easily become a part of. The downside to doing it that way is that this codebase is being ditched soon in favour of 1.0.0 which is built of off bf 4.3, in that case this would be conflicting with the normal FF which bf has. So just some simple way we can implement this is totally fine by me. Giving it a better name i'm find with as well, it just needs to have a distinction from the FF which betaflight uses which is the derivative of setpoint while this is proportional to the setpoint. So it'll be tuned differently.